### ISSUE TITLE
Unable to load local vector drawables

### ISSUE THREAD

**Issue by AndreaCioccarelli** [coil-kt/coil#86]
**Describe the bug**
I have recently switched from Glide to Coil as image loading library, but after having implemented Coil just some images are loaded, the ones that are loaded by URL, while vector drawables are not displayed at all, even though I'm trying to strictly follow the documentation

**To Reproduce**
Create a vector drawable, place it under /res/drawables, then 

```kotlin
val icon = layout.find<ImageView>(R.id.status_icon)
icon.load(R.drawable.status_ok)
```

**Expected behavior**
The icon should be displayed correctly
Coil Version 0.6.1

**Comment by colinrtwhite** [comment]
Thanks for the report. There are two main issues with vector drawables in `0.6.1` at the moment which should be fixed by #75 and #89. Both fixes will be available in `0.7.0` which will likely be released later today.

**Comment by AndreaCioccarelli** [comment]
Good to hear! I'll check out if the update works for me and report here any news.
